api/query/cache/index/aggregator.go

```diff
@@ -32,7 +33,26 @@ func (a *indexAggregator) Add(t TestID) error {
 			return err
 		}
 
-		r = query.SearchResult{Test: name, LegacyStatus: nil}
+		r = query.SearchResult{
+			Test:         name,
+			LegacyStatus: nil,
+			Interop:      nil,
+		}
+	}
+
+	if a.interopFormat {
+		if r.Interop == nil {
+			r.Interop = make([]int, len(a.rus)+1)
+		}
+		passing := 0
+		for _, ru := range a.rus {
```

Comment:
I see that the naming is not new, but in this context even knowing that the type is `[]RunID` I can't guess what `rus` or `ru` means. Run... unit? If it can be renamed to something more transparent, doing that here on or followup would be nice.

Reply:
Totally agree - no idea what ru was.